lib/controller/validate/show.js

```diff
@@ -0,0 +1,10 @@
+const validate = (showEvent) => {
```

Comment:
There is no validation logic here. Are you planning to add more later?

Reply:
Yes, sir. The status command had not validation.